### DISCUSSION TITLE
Maven package for a private repo.

### DISCUSSION THREAD

**Discussion by SwordOfSouls** [community/community#27735] [Actions] [answered]
Hey! 
I am attempting to use Github Packages to host a maven server for one of my artifacts and am running into some issues. [This is my settings.xml](https://www.toptal.com/developers/hastebin/xesutohuge.xml). I am pushing the artifact like this: https://www.toptal.com/developers/hastebin/abuvuwotem.xml, 
![image](https://user-images.githubusercontent.com/82922881/181981135-8d4bbb93-810d-43e8-95e1-85d3f00fddb3.png). 
When I try to use the API in another repo like [this](https://www.toptal.com/developers/hastebin/johudosora.xml), 
![image](https://user-images.githubusercontent.com/82922881/181982950-40d58846-305f-4b80-adfa-9ebf9128623d.png). 
I get this error: `Could not find artifact org.swordofsouls:UltimateGUI:pom:4.0.1 in central (https://repo1.maven.org/maven2)`. 
There are proceeded requests to the maven server before I get this error: `Could not find artifact org.swordofsouls:UltimateGUI:pom:4.0.1 in central (https://repo1.maven.org/maven2)`. 

Any help is greatly appreciated, thanks!

**Comment by SwordOfSouls** [comment]
bump

**Comment by SwordOfSouls** [comment, marked as answer]
solved

**Reply by laughedelic** [reply]
How did you solve it? Or was there a fix on the GitHub side?

**Reply by SwordOfSouls** [reply]
Hey! Ended up not using GitHubs maven system.
